aci_interface_policies.tf

```diff
@@ -238,17 +241,18 @@ locals {
 module "aci_leaf_interface_configuration_sub" {
   source = "./modules/terraform-aci-interface-configuration"
 
-  for_each          = { for int in local.new_leaf_subinterface_configuration : int.key => int if local.modules.aci_interface_configuration && var.manage_interface_policies }
-  node_id           = each.value.node_id
-  module            = each.value.module
-  port              = each.value.port
-  sub_port          = each.value.sub_port
-  policy_group_type = each.value.policy_group_type
-  policy_group      = each.value.policy_group
-  fex_id            = each.value.fex_id
-  description       = each.value.description
-  shutdown          = each.value.shutdown
-  role              = each.value.role
+  for_each                   = { for int in local.new_leaf_subinterface_configuration : int.key => int if local.modules.aci_interface_configuration && var.manage_interface_policies }
+  node_id                    = each.value.node_id
+  module                     = each.value.module
+  port                       = each.value.port
+  sub_port                   = each.value.sub_port
+  policy_group_type          = each.value.policy_group_type
+  policy_group               = each.value.policy_group
+  fex_id                     = each.value.fex_id
+  description                = each.value.description
+  shutdown                   = each.value.shutdown
+  role                       = each.value.role
+  port_channel_member_policy = try("${each.value.port_channel_member_policy}${local.defaults.apic.access_policies.interface_policies.port_channel_member_policies.name_suffix}", "")
```

Comment:
Is it supported on sub-interfaces?

Reply:
well, its not supported on 5.2 but its supported starting 6.0